### PR TITLE
Prepare 2.27.0

### PR DESCRIPTION
**Pull request by gre**
# 2.26.1 -> 2.27.0: What changed?

non exhaustive list of things that changes and that are interesting to test.

**see commits for full list.**

## Portfolio rework, All Time Range

We have fixed bugs and improve the performance of the balance history calculation on Ledger Live. There are still future improvement to perform on this topic but we believe the present work is stable enough.

That said, it's a big part of our app and help is appreciated to test it with your accounts, there are many pitfalls and edge cases depending on what you did with your accounts so it's never easy to cover it all.

Capture d’écran 2021-05-03 à 20 08 50Capture d’écran 2021-05-03 à 20 08 41

## Fee section rework (Ethereum + Bitcoin & altcoins)

There is an unification of Fees UI that simplify things for many users. You can still go "Advanced" to have more advanced fields.

Capture d’écran 2021-05-03 à 20 13 16Capture d’écran 2021-05-03 à 20 13 23

and Ethereum (today's fees hurt)

Capture d’écran 2021-05-03 à 20 14 43Capture d’écran 2021-05-03 à 20 14 50

## Allow inline app installation

In Settings > Experimental, we have a new feature that we will eventually enable for all:

Capture d’écran 2021-05-03 à 19 00 56

It allows to automatically get the require apps installed in middle of flows, like **Add Accounts** or **Send** flows. It's a major step forward for a better UX on Ledger Live. You do not need to worry anymore about having the apps before doing a Send flow, it will be installed in place, on need. *The only limitation is when you reach no more size on your device and you will still need to go to Manager in that case to make some.*

Capture d’écran 2021-05-03 à 19 01 23Capture d’écran 2021-05-03 à 19 01 30

**This works very well in combination of Nano S 2.0 firmware, feel free to test it with! We are looking for feedbacks for that feature but it's planned to enable it on the release that will follow.**

## DASH v3 explorer

We fixed issues on dash by switching to v3 explorers. 

## ERC20 update

```
+| Ethereum | 10SET | 0x7FF4169a6B5122b664c51c95727d87750eC07c84 | 10Set Token |  | ethereum/erc20/10set_token |
+| Ethereum | ALICE | 0xAC51066d7bEC65Dc4589368da368b212745d63E8 | ALICE |  | ethereum/erc20/alice |
-| Ethereum | AXPR | 0xC39E626A04C5971D770e319760D7926502975e47 | AXPR (old) | countervalues disabled | ethereum/erc20/axpr_old |
+| Ethereum | AXPR | 0xC39E626A04C5971D770e319760D7926502975e47 | AXPR (old) | countervalues disabled | ethereum/erc20/axpire_old |
+| Ethereum | ADCO | 0xB6c3DC857845a713d3531cea5ac546F6767992F4 | Advertise Coin |  | ethereum/erc20/advertise_coin |
+| Ethereum | ARNX | 0x0C37Bcf456bC661C14D596683325623076D7e283 | Aeron |  | ethereum/erc20/aeron |
+| Ethereum | MATTER | 0x1C9491865a1DE77C5b6e19d2E6a5F1D7a6F2b25F | Antimatter Finance Governance Token |  | ethereum/erc20/antimatter_finance_governance_token |
+| Ethereum | ARMOR | 0x1337DEF16F9B486fAEd0293eb623Dc8395dFE46a | Armor |  | ethereum/erc20/armor |
+| Ethereum | AUDIO | 0x18aAA7115705e8be94bfFEBDE57Af9BFc265B998 | Audius |  | ethereum/erc20/audius |
-| Ethereum | AWX | 0x54E46cC89588318e3964ca2c1Be94Db9D5cA3dFb | AurusCoin |  | ethereum/erc20/auruscoin |
+| Ethereum | AWX | 0x54E46cC89588318e3964ca2c1Be94Db9D5cA3dFb | AurusCoin (old) | countervalues disabled | ethereum/erc20/auruscoin_old |
+| Ethereum | AWX | 0x1EFdfC6146cAd8909817284AE99325EF1cAF623e | AurusDeFi |  | ethereum/erc20/aurusdefi |
-| Ethereum | NIO | 0x5554e04e76533E1d14c52f05beEF6c9d329E1E30 | Autonio |  | ethereum/erc20/autonio |
+| Ethereum | NIOX | 0xc813EA5e3b48BEbeedb796ab42A30C5599b01740 | Autonio |  | ethereum/erc20/autonio |
+| Ethereum | NIO | 0x5554e04e76533E1d14c52f05beEF6c9d329E1E30 | Autonio (old) | countervalues disabled | ethereum/erc20/autonio_old |
+| Ethereum | MARK | 0x67c597624B17b16fb77959217360B7cD18284253 | Benchmark |  | ethereum/erc20/benchmark |
+| Ethereum | BLES | 0xE796d6ca1ceb1b022EcE5296226BF784110031Cd | Blind Boxes Token |  | ethereum/erc20/blind_boxes_token |
+| Ethereum | CONV | 0xc834Fa996fA3BeC7aAD3693af486ae53D8aA8B50 | Convergence |  | ethereum/erc20/convergence |
+| Ethereum | USDK3CRV | 0x97E2768e8E73511cA874545DC5Ff8067eB19B787 | Curve fi USDK 3Crv |  | ethereum/erc20/curve_fi_usdk_3crv |
+| Ethereum | DAFI | 0xFc979087305A826c2B2a0056cFAbA50aad3E6439 | DAFI Token |  | ethereum/erc20/dafi_token |
+| Ethereum | DEXA | 0x725440512cb7b78bF56B334E50e31707418231CB | DEXA COIN |  | ethereum/erc20/dexa_coin |
+| Ethereum | DSLA | 0x3aFfCCa64c2A6f4e3B6Bd9c64CD2C969EFd1ECBe | DSLA |  | ethereum/erc20/dsla |
+| Ethereum | DDIM | 0xFbEEa1C75E4c4465CB2FCCc9c6d6afe984558E20 | DuckDaoDime |  | ethereum/erc20/duckdaodime |
+| Ethereum | ERN | 0xBBc2AE13b23d715c30720F079fcd9B4a74093505 | EthernityChain ERN Token |  | ethereum/erc20/ethernitychain_ern_token |
+| Ethereum | ID | 0xEBd9D99A3982d547C5Bb4DB7E3b1F9F14b67Eb83 | Everest ID |  | ethereum/erc20/everest_id |
+| Ethereum | XED | 0xee573a945B01B788B9287CE062A0CFC15bE9fd86 | Exeedme |  | ethereum/erc20/exeedme |
+| Ethereum | FXS | 0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0 | Frax Share |  | ethereum/erc20/frax_share |
+| Ethereum | FWB | 0x35bD01FC9d6D5D81CA9E055Db88Dc49aa2c699A8 | Friends With Benefits Pro |  | ethereum/erc20/friends_with_benefits_pro |
+| Ethereum | FUSE | 0x970B9bB2C0444F5E81e9d0eFb84C8ccdcdcAf84d | Fuse Token |  | ethereum/erc20/fuse_token |
+| Ethereum | GHX | 0x728f30fa2f100742C7949D1961804FA8E0B1387d | GamerCoin |  | ethereum/erc20/gamercoin |
+| Ethereum | AUS | 0x171f9cFc136f2B2aaA148fcC6b660a2029baB048 | Gold Standard |  | ethereum/erc20/gold_standard |
+| Ethereum | GDAO | 0x515d7E9D75E2b76DB60F8a051Cd890eBa23286Bc | Governor |  | ethereum/erc20/governor |
-| Ethereum | HNT | 0x08Abae9AF6713aC141D85e0b6ad825bb85F39220 | Helium |  | ethereum/erc20/helium |
+| Ethereum | HNT | 0x08Abae9AF6713aC141D85e0b6ad825bb85F39220 | Helium | delisted, countervalues disabled | ethereum/erc20/helium |
+| Ethereum | HEZ | 0xEEF9f339514298C6A857EfCfC1A762aF84438dEE | Hermez Network Token |  | ethereum/erc20/hermez_network_token |
-| Ethereum | HYDRO | 0xEBBdf302c940c6bfd49C6b165f457fdb324649bc | Hydro |  | ethereum/erc20/hydro |
+| Ethereum | HYDRO | 0xEBBdf302c940c6bfd49C6b165f457fdb324649bc | Hydro | countervalues disabled | ethereum/erc20/hydro |
-| Ethereum | IDEA | 0x814CAfd4782d2e728170FDA68257983F03321c58 | IDEA Token |  | ethereum/erc20/idea_token |
+| Ethereum | IDEA | 0x814CAfd4782d2e728170FDA68257983F03321c58 | IDEA Token | countervalues disabled | ethereum/erc20/idea_token |
+| Ethereum | IDEA | 0x5d3a4F62124498092Ce665f865E0b38fF6F5FbEa | IDEAOLOGY |  | ethereum/erc20/ideaology |
+| Ethereum | XIV | 0x44f262622248027f8E2a8Fb1090c4Cf85072392C | INVERSE |  | ethereum/erc20/inverse |
+| Ethereum | INX | 0xBBC7f7A6AADAc103769C66CBC69AB720f7F9Eae3 | INX Token |  | ethereum/erc20/inx_token |
+| Ethereum | IQQ | 0x68a9d92Fe19399FEEBEd6A9a0980a7ea7638074C | IQONIQ |  | ethereum/erc20/iqoniq |
+| Ethereum | KEX | 0x16980b3B4a3f9D89E33311B5aa8f80303E5ca4F8 | KIRA Network |  | ethereum/erc20/kira_network |
+| Ethereum | KTN | 0x491E136FF7FF03E6aB097E54734697Bb5802FC1C | Kattana |  | ethereum/erc20/kattana |
+| Ethereum | KONO | 0x850aAB69f0e0171A9a49dB8BE3E71351c8247Df4 | Konomi |  | ethereum/erc20/konomi |
+| Ethereum | LABS | 0x8b0E42F366bA502d787BB134478aDfAE966C8798 | LABS Group |  | ethereum/erc20/labs_group |
+| Ethereum | LINA | 0xC05d14442A510De4D3d71a3d316585aA0CE32b50 | LINA |  | ethereum/erc20/lina |
+| Ethereum | LTX | 0xa393473d64d2F9F026B60b6Df7859A689715d092 | Lattice Token |  | ethereum/erc20/lattice_token |
+| Ethereum | LINA | 0x3E9BC21C9b189C09dF3eF1B824798658d5011937 | Linear Token |  | ethereum/erc20/linear_token |
-| Ethereum | LIT | 0x763Fa6806e1acf68130D2D0f0df754C93cC546B2 | Lition |  | ethereum/erc20/lition |
+| Ethereum | LIT | 0x763Fa6806e1acf68130D2D0f0df754C93cC546B2 | Lition | countervalues disabled | ethereum/erc20/lition |
+| Ethereum | MDX | 0x947AEb02304391f8fbE5B25D7D98D649b57b1788 | MANDALA EXCHANGE TOKEN |  | ethereum/erc20/mandala_exchange_token |
+| Ethereum | MIMO | 0x90B831fa3Bebf58E9744A14D638E25B4eE06f9Bc | MIMO Parallel Governance Token |  | ethereum/erc20/mimo_parallel_governance_token |
+| Ethereum | MX | 0x11eeF04c884E24d9B7B4760e7476D06ddF797f36 | MX Token |  | ethereum/erc20/mx_token |
-| Ethereum | MVP | 0x8a77e40936BbC27e80E9a3F526368C967869c86D | Merculet |  | ethereum/erc20/merculet |
+| Ethereum | MVP | 0x432A2C54de2DDE941A36d2EB8c424Ed666F74Aef | Merculet |  | ethereum/erc20/merculet |
+| Ethereum | MVP | 0x8a77e40936BbC27e80E9a3F526368C967869c86D | Merculet (old) | countervalues disabled | ethereum/erc20/merculet_old |
+| Ethereum | MTHD | 0x84bA4aEcfDe39D69686a841BAb434C32d179a169 | Method |  | ethereum/erc20/method |
+| Ethereum | MBL | 0xB879DA8b24c9b8685dE8526cF492E954f165D74b | MovieBloc |  | ethereum/erc20/moviebloc |
+| Ethereum | MUSE | 0xB6Ca7399B4F9CA56FC27cBfF44F4d2e4Eef1fc81 | Muse |  | ethereum/erc20/muse |
+| Ethereum | OXY | 0x869b1F57380aE501d387b19262EFD3C0Eb7501b0 | Oxycoin |  | ethereum/erc20/oxycoin |
+| Ethereum | POP | 0x5D858bcd53E085920620549214a8b27CE2f04670 | POP Network Token |  | ethereum/erc20/pop_network_token |
+| Ethereum | PAR | 0x1BeEF31946fbbb40B877a72E4ae04a8D1A5Cee06 | Parachute |  | ethereum/erc20/parachute |
+| Ethereum | PHNX | 0x38A2fDc11f526Ddd5a607C1F251C065f40fBF2f7 | PhoenixDAO |  | ethereum/erc20/phoenixdao |
+| Ethereum | CVR | 0x3C03b4EC9477809072FF9CC9292C9B25d4A8e6c6 | PolkaCover |  | ethereum/erc20/polkacover |
-| Ethereum | NPXS | 0xA15C7Ebe1f07CaF6bFF097D8a589fb8AC49Ae5B3 | Pundi X Token |  | ethereum/erc20/pundi_x_token |
+| Ethereum | PUNDIX | 0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38 | Pundi X Token |  | ethereum/erc20/pundi_x_token |
+| Ethereum | NPXS | 0xA15C7Ebe1f07CaF6bFF097D8a589fb8AC49Ae5B3 | Pundi X Token (old) | countervalues disabled | ethereum/erc20/pundi_x_token_old |
+| Ethereum | RLY | 0xf1f955016EcbCd7321c7266BccFB96c68ea5E49b | Rally |  | ethereum/erc20/rally |
+| Ethereum | BUNNY | 0x3Ea50B7Ef6a7eaf7E966E2cb72b519C16557497c | Rocket Bunny |  | ethereum/erc20/rocket_bunny |
+| Ethereum | SHIB | 0x95aD61b0a150d79219dCF64E1E6Cc01f0B64C4cE | SHIBA INU |  | ethereum/erc20/shiba_inu |
+| Ethereum | SI | 0xD23Ac27148aF6A2f339BD82D0e3CFF380b5093de | SIREN |  | ethereum/erc20/siren |
+| Ethereum | SPI | 0x9B02dD390a603Add5c07f9fd9175b7DABE8D63B7 | Shopping.io |  | ethereum/erc20/shopping.io |
+| Ethereum | AGS | 0x843C9AF34F698618F90C898E3967278a260c8d9A | Silver Standard |  | ethereum/erc20/silver_standard |
-| Ethereum | SMT | 0x55F93985431Fc9304077687a35A1BA103dC1e081 | SmartMesh |  | ethereum/erc20/smartmesh |
+| Ethereum | SMT | 0x55F93985431Fc9304077687a35A1BA103dC1e081 | SmartMesh | countervalues disabled | ethereum/erc20/smartmesh |
+| Ethereum | RETH2 | 0x20BC832ca081b91433ff6c17f85701B6e92486c5 | StakeWise Reward ETH2 |  | ethereum/erc20/stakewise_reward_eth2 |
+| Ethereum | SETH2 | 0xFe2e637202056d30016725477c5da089Ab0A043A | StakeWise Staked ETH2 |  | ethereum/erc20/stakewise_staked_eth2 |
+| Ethereum | SMT | 0xc761c8Dc05Ae52a8a785665E528ddbb00C098AD1 | Sun Money Token |  | ethereum/erc20/sun_money_token |
+| Ethereum | SUTER | 0xAA2ce7Ae64066175E0B90497CE7d9c190c315DB4 | Suterusu |  | ethereum/erc20/suterusu |
+| Ethereum | SYLO | 0xf293d23BF2CDc05411Ca0edDD588eb1977e8dcd4 | Sylo |  | ethereum/erc20/sylo |
+| Ethereum | TIKTOK | 0xF4edA77f0B455A12f3eb44F8653835f377e36b76 | TIKTOK COIN |  | ethereum/erc20/tiktok_coin |
+| Ethereum | TOWER | 0x1C9922314ED1415c95b9FD453c3818fd41867d0B | TOWER |  | ethereum/erc20/tower |
+| Ethereum | TRXC | 0xaD5Fe5B0B8eC8fF4565204990E4405B2Da117d8e | TRONCLASSIC |  | ethereum/erc20/tronclassic |
+| Ethereum | TROY | 0x4574562E9310a94F9Ca962bd23168d8a06875b1A | TROY |  | ethereum/erc20/troy |
+| Ethereum | TAP | 0x7f1F2D3dFa99678675ECE1C243d3f7bC3746db5D | Tapmydata |  | ethereum/erc20/tapmydata |
+| Ethereum | TFT | 0xaeF4F02E31CdbF007f8D98da4aE365188A0E9eCC | The Famous Token |  | ethereum/erc20/the_famous_token |
+| Ethereum | UBX | 0xF5B5eFc906513b4344EbAbCF47A04901f99F09f3 | UBX |  | ethereum/erc20/ubx |
+| Ethereum | TRADE | 0x6F87D756DAf0503d08Eb8993686c7Fc01Dc44fB1 | UniTrade |  | ethereum/erc20/unitrade |
+| Ethereum | UDO | 0xea3983Fc6D0fbbC41fb6F6091f68F3e08894dC06 | Unido |  | ethereum/erc20/unido |
+| Ethereum | UCO | 0x8a3d77e9d6968b780564936d15B09805827C21fa | UnirisToken |  | ethereum/erc20/uniristoken |
+| Ethereum | MARSH | 0x5a666c7d92E5fA7Edcb6390E4efD6d0CDd69cF37 | UnmarshalToken |  | ethereum/erc20/unmarshaltoken |
+| Ethereum | UBXT | 0x8564653879a18C560E7C0Ea0E084c516C62F5653 | UpBots |  | ethereum/erc20/upbots |
+| Ethereum | VISR | 0xF938424F7210f31dF2Aee3011291b658f872e91e | VISOR |  | ethereum/erc20/visor |
+| Ethereum | ACDC | 0xFc44EC51C80e35A87Bc2140299B1636eC83DFb04 | VOLT |  | ethereum/erc20/volt |
+| Ethereum | WHALE | 0x9355372396e3F6daF13359B7b607a3374cc638e0 | WHALE |  | ethereum/erc20/whale |
+| Ethereum | WILC | 0xC98a910eDE52E7d5308525845F19e17470DbCCF7 | Wrapped ILCOIN |  | ethereum/erc20/wrapped_ilcoin |
+| Ethereum | WLOTTO | 0x37605EeF354fA8d1Cca7B923Da7e451269C74FbC | Wrapped Lotto Token |  | ethereum/erc20/wrapped_lotto_token |
+| Ethereum | XLAB | 0x8c4E7f814d40f8929F9112C5D09016F923d34472 | XCELTOKEN PLUS |  | ethereum/erc20/xceltoken_plus |
+| Ethereum | XEND | 0xE4CFE9eAa8Cdb0942A80B7bC68fD8Ab0F6D44903 | XEND |  | ethereum/erc20/xend |
+| Ethereum | YCC | 0x37E1160184F7dD29f00b78C050Bf13224780b0B0 | Yuan Chain New |  | ethereum/erc20/yuan_chain_new |
+| Ethereum | ZORA | 0xD8E3FB3b08eBA982F2754988d70D57eDc0055ae6 | Zoracles |  | ethereum/erc20/zoracles |
+| Ethereum | DHT | 0xca1207647Ff814039530D7d35df0e1Dd2e91Fa84 | dHedge DAO Token |  | ethereum/erc20/dhedge_dao_token |
+| Ethereum | HOGE | 0xfAd45E47083e4607302aa43c65fB3106F1cd7607 | hoge finance |  | ethereum/erc20/hoge_finance |
+| Ethereum | K21 | 0xB9d99C33eA2d86EC5eC6b8A4dD816EBBA64404AF | k21 kanon art |  | ethereum/erc20/k21_kanon_art |
+| Tron | KLV | TVj7RNVHy6thbM7BWdSe9G6gXwKhjhdNZS | Klever | countervalues disabled | tron/trc20/TVj7RNVHy6thbM7BWdSe9G6gXwKhjhdNZS |
+| Tron | SEED | TBwoSTyywvLrgjSgaatxrBhxt3DGpVuENh | SEED | countervalues disabled | tron/trc20/TBwoSTyywvLrgjSgaatxrBhxt3DGpVuENh |
```

### CODE REVIEW

**Review by JunichiSugiura — CHANGES_REQUESTED** [review]
I was gonna scanning through this pr since these codes are already reviewed but i can't ignore this.

File: src/renderer/components/BalanceInfos/index.js [hunk 1]
```diff
@@ -38,28 +30,17 @@ type Props = {
...
-  ...boxProps
-}: Props) {
+export function BalanceDiff({ totalBalance, valueChange, unit, isAvailable, ...boxProps }: Props) {
   if (!isAvailable) return null;
 
   return (
     <Box horizontal {...boxProps}>
       <Box horizontal alignItems="center" style={{ lineHeight: 1.2, fontSize: 20 }}>
-        {valueChange.percentage && (
+        {valueChange.percentage && valueChange.percentage && (
```
Comment (JunichiSugiura) [inline src/renderer/components/BalanceInfos/index.js:39]: I mean this. I'm gonna push a fix shortly.

**Comment by JunichiSugiura** [comment]
Did we change the mock data for DAI countervalue? I'm wondering why the price not shown in asset allocation list.

**Review by JunichiSugiura — APPROVED** [review]

**Comment by gre** [comment]
WTF
